### DISCUSSION TITLE
Accessing server ID as a client

### DISCUSSION THREAD

**Discussion by pbar1** [Eugeny/russh#336] [Q&A] [answered]
Using an SSH client, I'd like to access the server's ID string, ie:
```
♪ nc example.lan 22
SSH-2.0-OpenSSH_9.8
```

This doesn't seem possible with `russh`, unless I'm missing something? Is there a way? I mistakenly tried implementing `auth_banner` with a oneshot channel thinking it was what I wanted, but that is actually the server's login banner that's displayed on a successful login attempt.

**Comment by qsantos** [comment, marked as answer]
I think [`client::Session::remote_sshid`](https://docs.rs/russh/latest/russh/client/struct.Session.html#method.remote_sshid) is what you are looking for.

**Reply by pbar1** [reply]
Exactly is, thank you!
